src/lib/__tests__/InstantSearch-test.tsx

```diff
@@ -2038,7 +2038,7 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/instantsear
 
   test('with function form sets indices state', async () => {
     const searchClient = createSearchClient();
-    const search = new InstantSearch({
+    const search = new InstantSearch<UiState>({
```

Comment:
This is needed because initialUiState seems to "fill in" the generic. That makes sense, but in this case it makes things required that aren't actually required. Not sure how to solve that though! This already was the case before the PR, but became visible due to usage of TUiState in setUiState

Reply:
https://github.com/algolia/instantsearch.js/pull/5061